app/views/components/_download-link.html.erb

```diff
@@ -0,0 +1,9 @@
+<%
+  download_link_text ||= "Download File"
+%>
+
+<div class="app-c-download-link">
```

Comment:
When seeing the focus state it looks oddly big, I wonder if that min-height is what's making it bigger than you'd expect?